testing/test_runner.py

```diff
@@ -336,10 +336,9 @@ def test_method(self):
         assert reps[2].failed
         assert reps[2].when == "teardown"
         assert reps[2].longrepr.reprcrash.message in (
-            # python3 error
             "TypeError: teardown_method() missing 2 required positional arguments: 'y' and 'z'",
-            # python2 error
-            "TypeError: teardown_method() takes exactly 4 arguments (2 given)",
```

Comment:
need to keep the python2 error message in this sequence since 4.6.x still supports python 2.x

Reply:
on it